src/Features/Core/Portable/Completion/MatchPriority.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Completion
+{
+    /// <summary>
+    /// An additional hint to the matching algorithm that can
+    /// augment or override the existing text-based matching.
+    /// </summary>
+    public static class MatchPriority
+    {
+        /// <summary>
+        /// The matching algorithm should give this item no special treatment.
+        /// 
+        /// Ordinary <see cref="CompletionProvider"/>s typically specify this.
+        /// </summary>
+        public const int Default = 0;
```

Comment:
@CyrusNajmabadi I had to make `Default` a const because we use it as the default value of an optional argument in lots of places.